akka-distributed-data/src/main/scala/akka/cluster/ddata/ORSet.scala

```diff
@@ -201,8 +210,13 @@ object ORSet {
 @SerialVersionUID(1L)
 final class ORSet[A] private[akka] (
   private[akka] val elementsMap: Map[A, ORSet.Dot],
-  private[akka] val vvector:     VersionVector)
-  extends ReplicatedData with ReplicatedDataSerialization with RemovedNodePruning with FastMerge {
+  private[akka] val vvector:     VersionVector,
+  private[akka] val _delta:      Option[ORSet[A]]  = None,
```

Comment:
Maybe this could be a `List`? Not necessarily for `ORSet` but for other CRDT types - to have this possibility.